WordPress/src/main/java/org/wordpress/android/ui/people/PeopleManagementActivity.java

```diff
@@ -120,4 +124,113 @@ public void onPersonSelected(Person person) {
             fragmentTransaction.commit();
         }
     }
+
+    @Override
+    public void onRoleChanged(long personID, int localTableBlogId, String newRole) {
+        Person person = PeopleTable.getPerson(personID, localTableBlogId);
+        if (person == null || newRole == null || newRole.equalsIgnoreCase(person.getRole())) {
+            return;
+        }
+        PeopleUtils.updateRole(person.getBlogId(), person.getPersonID(), newRole, localTableBlogId,
+                new PeopleUtils.UpdateUserCallback() {
+            @Override
+            public void onSuccess(Person person) {
+                PeopleTable.save(person);
+                refreshOnScreenFragmentDetails();
+            }
+
+            @Override
+            public void onError() {
+                ToastUtils.showToast(PeopleManagementActivity.this,
+                        R.string.error_update_role,
+                        ToastUtils.Duration.LONG);
+            }
+        });
+    }
+
+    private void confirmRemovePerson() {
+        if (mSelectedPerson == null) {
+            return;
+        }
+
+        AlertDialog.Builder builder = new AlertDialog.Builder(this, R.style.Calypso_AlertDialog);
+        builder.setTitle(getString(R.string.person_remove_confirmation_title, mSelectedPerson.getDisplayName()));
+        builder.setMessage(getString(R.string.person_remove_confirmation_message, mSelectedPerson.getDisplayName()));
+        builder.setNegativeButton(R.string.cancel, null);
+        builder.setPositiveButton(R.string.remove, new DialogInterface.OnClickListener() {
+            @Override
+            public void onClick(DialogInterface dialog, int which) {
+                removeSelectedPerson();
+            }
+        });
+        builder.show();
+    }
+
+    private void removeSelectedPerson() {
+        if (mSelectedPerson == null) {
+            return;
+        }
+        PeopleUtils.removePerson(mSelectedPerson.getBlogId(),
+                mSelectedPerson.getPersonID(),
+                mSelectedPerson.getLocalTableBlogId(),
+                new PeopleUtils.RemoveUserCallback() {
+                    @Override
+                    public void onSuccess(long personID, int localTableBlogId) {
+                        // remove the person from db, navigate back to list fragment and refresh it
+                        Person person = PeopleTable.getPerson(personID, localTableBlogId);
+                        String text;
+                        if (person != null) {
+                            PeopleTable.deletePerson(personID, localTableBlogId);
+                            text = getString(R.string.person_removed, person.getUsername());
```

Comment:
Not sure if this is by design but, since we are asking for remove confirmation using the Person's display name, I think it makes sense to use the same in the Toast as well. Cheers!